### PR TITLE
Lesson 3

### PR DESCRIPTION
**Pull request by sergey87chuvasov**
home task 3 done, awaiting feedback

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,186 @@
...
+      </tr>
+      <tr>
+        <td>&lt;tfoot&gt;</td>
+      <td>Groups the footer content in a table</td>
+      </tr>
+    </table>
+    <br/>
+    <br/>
+    <br/>
+    <form action:'/' method:'GET'>
```
Comment (BubentsovKirill) [inline index.html:129]: Align the indents in the form.

File: index.html [hunk 2]
```diff
@@ -1,15 +1,186 @@
...
+    <br/>
+      <h4>Comment</h4>
+        <textarea name="textBig" cols="30" rows="4" placeholder="Comment" maxlength="500">
+        </textarea>
+    <br/>
+      <h4>What does HTML stand for?</h4>
+        <select name="answer">
+          <option>Hyper Text Markup Language</option>
+          <option>Hyper Toll Markup Language</option>
+          <option>Hyper Thron Markup Language</option>
```
Comment (BubentsovKirill) [inline index.html:177]: `option` tags should have `value` attr.

File: index.html [hunk 3]
```diff
@@ -1,15 +1,186 @@
...
+    <br/>
+        <input type="checkbox" id="answer3" name="test3" value="3" />
+        <label for="answer3">Hyper Text Markup Language</label>
+    <br/>
+      <h4>What does HTTP stand for?</h4>
+        <input type="radio" name="answer1" value="1"/>Hyper Text Translate Protokol
+    <br/>
+        <input type="radio" name="answer2" value="2"/>Hyper Text Transmission Protokol
+    <br/>
+        <input type="radio" name="answer3" value="3"/>Hyper Text Transfer Protokol
```
Comment (BubentsovKirill) [inline index.html:167]: Text for inputs should be wrapped in `label`.
Inputs `radio` should have the same `name` attr.

File: index.html (hunk 1 above)
Comment (sergey87chuvasov) [reply index.html:129]: fixed indents in textarea

File: index.html (hunk 2 above)
Comment (sergey87chuvasov) [reply index.html:177]: added value attribute in tag option

File: index.html (hunk 3 above)
Comment (sergey87chuvasov) [reply index.html:167]: added label to text inputs, rename name atribute in radiobuttons group

**Pull request merged**